<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PartCreationWindow</class>
 <widget class="QDialog" name="PartCreationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Part details</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1" colspan="3">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="1">
      <widget class="QLabel" name="imageLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="sKULabel">
       <property name="text">
        <string>SKU</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="sKULineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="lookupButton">
         <property name="text">
          <string>Lookup..</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="supplierLabel">
       <property name="text">
        <string>Supplier</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="supplierLineEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="supplierURLLabel">
       <property name="text">
        <string>Supplier URL</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="mPNLabel">
       <property name="text">
        <string>MPN</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="mPNLineEdit"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="manufacturerLabel">
       <property name="text">
        <string>Manufacturer</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="manufacturerLineEdit"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Description Example</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="exampleLable">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="descriptionLabel">
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="descriptionLineEdit"/>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="eRPEGNavisionLabel">
       <property name="text">
        <string>ERP(e.g.navision)</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLineEdit" name="eRPLineEdit"/>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="datasheetURLLabel">
       <property name="text">
        <string>Datasheet URL</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLineEdit" name="datasheetLinkLineEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="datasheetlinkLabel">
         <property name="text">
          <string>llink</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="17" column="0">
      <widget class="QLabel" name="qtyLabel">
       <property name="text">
        <string>Qty</string>
       </property>
      </widget>
     </item>
     <item row="17" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSpinBox" name="qtySpinBox">
         <property name="maximum">
          <number>1000000000</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="qtyManyCheckbox">
         <property name="text">
          <string>Many</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLineEdit" name="supplierLinkLineEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="supplierlinkLinkLabel">
         <property name="text">
          <string>link</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="18" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="locationLineEdit"/>
       </item>
      </layout>
     </item>
     <item row="18" column="0">
      <widget class="QLabel" name="lotLabel">
       <property name="text">
        <string>Location</string>
       </property>
      </widget>
     </item>
     <item row="19" column="0" colspan="2">
      <widget class="QLabel" name="additionalparameterLabel">
       <property name="text">
        <string/>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="continueWithNextPartButton">
     <property name="text">
      <string>Continue With next part</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="scanbarcodeButton">
     <property name="text">
      <string>Scan Barcodes..</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="2">
    <widget class="QTreeWidget" name="treeWidget">
     <column>
      <property name="text">
       <string>Categories</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PartCreationWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PartCreationWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
